kibbeh/src/modules/search/SearchBarController.tsx

```diff
@@ -90,6 +90,7 @@ export const SearchBarController: React.FC<SearchControllerProps> = ({}) => {
                 ? t("components.search.placeholderShort")
                 : t("components.search.placeholder")
             }
+            isLoading={isLoading}
```

Comment:
I believe you can just use ES6 and write `isLoading` instead of ` isLoading={isLoading}`

Reply:
Doing that is making isLoading true always, I think we can only do that on objects.